app/views/admin/splits/_remote_kills.html.erb

```diff
@@ -0,0 +1,34 @@
+<article class="InfoCard sc-m-v--l">
+  <div class="InfoCard-titleContainer">
+    <h4>Remote kills</h4>
+  </div>
+  <hr class="InfoCard-divider">
+  <div class="InfoCard-description">
+    <% if split.remote_kills.any? %>
+      <table class="fs-VariantsTable DescriptionTable">
+        <thead>
+          <th>App</th>
+          <th>Reason</th>
+          <th>From version</th>
+          <th>To version</th>
+        </thead>
+        <tbody>
+          <% split.remote_kills.each do |remote_kill| %>
+            <tr>
+              <td><%= remote_kill.app.name %></td>
+              <td><%= remote_kill.reason %></td>
+              <td><%= remote_kill.first_bad_version %></td>
```

Comment:
`first_bad_version` is marked as [an arrary in the schema](https://github.com/Betterment/test_track/blob/master/db/schema.rb#L63:L63)... tbh, i'm not sure what that means, but do we need to handle it in some special way?

Reply:
Oh I think it's an array because it's a Semver value

Reply:
...but I don't think we have to handle it special because it's deserialized into a type called AppVersion

Reply:
Yup that was so PG could sort it for us in semver order.